diff --git a/lisp/progmodes/csharp-mode.el b/lisp/progmodes/csharp-mode.el
index 29325ab9632..35cfe12d7e6 100644
--- a/lisp/progmodes/csharp-mode.el
+++ b/lisp/progmodes/csharp-mode.el
@@ -759,8 +759,7 @@ csharp-ts-mode--font-lock-settings
    :override t
    :feature 'keyword
    `([,@csharp-ts-mode--keywords] @font-lock-keyword-face
-     (modifier) @font-lock-keyword-face
-     (this_expression) @font-lock-keyword-face)
+     (modifier) @font-lock-keyword-face)
 
    :language 'c-sharp
    :override t
